test/hotspot/jtreg/runtime/ErrorHandling/TestHeapDumpOnOutOfMemoryError.java

```diff
@@ -78,6 +78,7 @@ static void test(String type) throws Exception {
                 //  MaxMetaspaceSize=16M - ~12-15K classes - ~12sec runtime with all verifications
                 //  MaxMetaspaceSize=16M - ~12-15K classes - VerifyDependencies off - ~3seconds on ppc
                 "-XX:MaxMetaspaceSize=16m",
+                "-XX:+IgnoreUnrecognizedVMOptions",
                 "-XX:-VerifyDependencies",
```

Comment:
Instead of adding "-XX:+IgnoreUnrecognizedVMOptions", you can
check the JDK type like this:

```
String jdkType = System.getProperty("jdk.debug", "release");
boolean addNonReleaseOptions = false;
if (!jdkType.equals("release")) {
    addNonReleaseOptions = true;
}
```

and then only include the "-XX:-VerifyDependencies" option
when `addNonReleaseOptions` is true... I'm not sure how to
do optional parameters with ProcessTools.createJavaProcessBuilder().

Reply:
I didn't know about `Platform.isDebugBuild()`...